lib/charms/spark.py

```diff
@@ -1,30 +1,26 @@
+# python3 pylint: disable=c0111,E0611,e0401,E1123,c0301
+from subprocess import CalledProcessError
 from glob import glob
 from path import Path
-from subprocess import CalledProcessError
 
 import jujuresources
+from jujubigdata import utils
 from charmhelpers.core import hookenv
 from charmhelpers.core import host
-from charmhelpers.core import unitdata
 from charmhelpers.fetch.archiveurl import ArchiveUrlFetchHandler
-from jujubigdata import utils
-
 
 # Main Spark class for callbacks
 class Spark(object):
     def __init__(self, dist_config):
         self.dist_config = dist_config
+        print(self.dist_config)
```

Comment:
Thanks for pointing this out, this should be removed altogether, I added it for debugging.